engine/src/tangl/service38/controllers.py

```
@@ -199,45 +195,77 @@ def drop_story(
     )
     def get_user_info(self, user: User, **kwargs: Hash) -> UserInfo:
-        return super().get_user_info(user=user, **kwargs)
+        return UserInfo.from_user(user, **kwargs)
 
     @ApiEndpoint38.annotate(
         access_level=AccessLevel.USER,
+        method_type=MethodType.DELETE,
+        response_type=ResponseType.RUNTIME,
         binds=(ResourceBinding.USER,),
     )
     def drop_user(self, user: User, **kwargs: Hash) -> RuntimeInfo:
```

Ruff (0.15.1)

[warning] Unused method argument: `kwargs`

(ARG002)